src/middlewares/badSeach.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-else-return */
```

Comment:
There is no need for this middleware. Return the response in the search controller